<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Hp" failures="1" tests="5" name="Test(failed)" time="40.966" errors="0" timestamp="2024-12-18T18:44:47 IST">
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.143"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="1.775">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:49)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.152"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="1.831"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.145"/>
</testsuite> <!-- Test(failed) -->
